### DISCUSSION TITLE
Sintaxe mais próxima do VisuAlg

### DISCUSSION THREAD

**Discussion by rabelofh** [dgadelha/Portugol-Webstudio#365] [Ideias e sugestões]
### Descrição do problema

Saudações, é possível ajustar o Portugol Studio com a sintaxe mais prox do VisuAlg?

A sintaxe que dispõe padrão aproxima-se mais da linguagem C, e a sintaxe do visuAlg é um pouco mais simplificada.

### Código

```portugol
Algoritmo "semnome"
// Descrição   : Aqui você descreve o que o programa faz! (função)
// Autor(a)    : Nome do(a) aluno(a)
// Data atual  : 04/10/2024
Var
// Seção de Declarações das variáveis 


Inicio
// Seção de Comandos, procedimento, funções, operadores, etc... 


Fimalgoritmo
```

**Comment by Edsonsilva-RJ** [comment]
Quando comecei a usar o Portugol Web studio acabei estranhando um pouco a liguagem. 

Tive a ilusão que fizesse parte sa mesma iniciativa do VisualG

**Comment by IFMASTIGabriel** [comment]
Acho que a proposta do PS é justamente ficar o mais próximo da linguagem C para tentar facilitar a transição
